src/organisms/cards/SimplePolicyCard/__tests__/simple_policy_card.spec.js

```diff
@@ -14,19 +13,13 @@ describe('<SimplePolicyCard />', () => {
         price: 19.8,
         format: 'mo'
       },
-      onDetails: () => {},
+      onDetails: jest.fn(),
       compareCheckbox: {
-        onCompare: () => {},
+        onCompare: jest.fn(),
```

Comment:
you have an extra comma here that's showing up in the snapshot

Reply:
fixed snapshot